new mdx scopes and windows paths

**Describe the bug**

```
Error: ./node_modules/gatsby-mdx/loaders/mdx-scopes.js
Module not found: Error: Can't resolve 'C:Users
tablesDesktopmdx-issuegatsby-mdx-issue-react-undefined.cachegatsby-mdxmdx-scopes-dicd190d04cf5fa8ed9cc7cc3be786ee0.js' in 'C:\Users\btables\Desktop\mdx-issue\gatsby-mdx-issue-react-undefined'
```

**To Reproduce**

Any programmatic MDX (not `src/pages`) after the new mdx scopes was merged (0.3.2 would do it)

**Expected behavior**

Build doesn't fail on windows

**Additional context**
https://github.com/ChristopherBiscardi/gatsby-mdx/issues/229#issuecomment-451769495

--- a/packages/gatsby-mdx/loaders/mdx-scopes.js
+++ b/packages/gatsby-mdx/loaders/mdx-scopes.js
@@ -1,5 +1,6 @@
 const fs = require("fs");
 const path = require("path");
+const slash = require("slash");
 
 module.exports = () => {
   const files = fs.readdirSync("./.cache/gatsby-mdx/mdx-scopes-dir");
@@ -8,7 +9,7 @@ module.exports = () => {
     files
       .map(
         (file, i) =>
-          `const scope_${i} = require('${path.join(abs, file)}').default;`
+          `const scope_${i} = require('${slash(path.join(abs, file))}').default;`
       )
       .join("\n") +
     `export default {

